### PR TITLE
SRVCOM-1532: Add the Troubleshooting Serverless section

### PR DESCRIPTION
**Pull request by rh-max**
For versions: ?

Preview:

### CODE REVIEW

File: modules/serverless-rn-1-19-0.adoc [hunk 1]
```diff
@@ -30,31 +30,4 @@
...
-----
-
-** Alternatively, you can expose the podman API by using TCP:
-+
-[source,terminal]
-----
-$ podman system service --time=0 tcp:127.0.0.1:5534 &
-export DOCKER_HOST=tcp://127.0.0.1:5534
-----
+See the workaround in the xref:../serverless/serverless-troubleshooting.adoc#failure-to-build-a-function-by-using-podman[Troubleshooting section].
```
Comment (abrennan89) [inline modules/serverless-rn-1-19-0.adoc:33]: can't have an xref in a module afaik

File: modules/serverless-deprecated-removed-features.adoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
+[id="serverless-deprecated-removed-features_{context}"]
+= Deprecated and removed features
```
Comment (abrennan89) [inline modules/serverless-deprecated-removed-features.adoc:2]: wasn't this added in a different PR for release notse?

File: modules/serverless-rn-1-20-0.adoc [hunk 3]
```diff
@@ -0,0 +1,57 @@
+[id="serverless-rn-1-20-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.20.0
```
Comment (abrennan89) [inline modules/serverless-rn-1-20-0.adoc:2]: Confused about why this is in two different PRs - is this troubleshooting guide being pushed out for 1.21.0?

File: modules/serverless-troubleshooting-functions.adoc [hunk 4]
```diff
@@ -0,0 +1,42 @@
+[id="serverless-troubleshooting-functions_{context}"]
+= Troubleshooting functions
+
+== Failure to build a function by using podman
```
Comment (abrennan89) [inline modules/serverless-troubleshooting-functions.adoc:4]: needs intro paragraph and ID I think for the h2

File: _topic_maps/_topic_map.yml [hunk 5]
```diff
@@ -3276,6 +3276,9 @@ Topics:
 # Support
 - Name: Support
   File: serverless-support
+# Troubleshooting
+- Name: Troubleshooting
+  File: serverless-troubleshooting
```
Comment (abrennan89) [inline _topic_maps/_topic_map.yml:3281]: I'd maybe put this inside the Support or Reference section, wdyt?

**Review by abrennan89 — CHANGES_REQUESTED** [review]
It looks like the relnotes content is still duplicated here, will this be ready for the release tomorrow?

**Comment by rh-max** [comment]
> It looks like the relnotes content is still duplicated here, will this be ready for the release tomorrow?

As discussed in the channel, Troubleshooting doesn't go out in 1.20, so I'll close this. This duplicated the 1.20 release notes PR for the goal of getting some material for illustrative purposes for Troubleshooting; it's outdated now.

**Pull request closed without merging**